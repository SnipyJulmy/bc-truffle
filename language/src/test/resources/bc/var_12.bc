define void f() {
  213
}
f()
define void f() {
 452
}
f()
define void g() {
 452
}
g()
f()
define void f() {
 458732
}
f()
halt
