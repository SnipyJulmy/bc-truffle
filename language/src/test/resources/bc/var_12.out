213
452
452
452
458732
